bin/make-snake

```diff
@@ -96,6 +96,17 @@ function colourEcho
     colour_echo "$1"
 }
 
+function ensureContentDir
+{
```

Comment:
you could add something like:
`mkdir -p ~/Snake-content` 
which I think it a no-op if the folder already exists here